services.yaml

```diff
@@ -3517,6 +3582,54 @@ definitions:
             type: "string"
             example: "label=sample_label_name"
         type: "object"
+  PublishMetricsSettings:
+    type: object
+    properties:
+      enable:
+        type: boolean
+        example: true
+      hostname:
+        type: string
+        description: The hostname and (optional) port of your StatsD server.
+        example: 127.0.0.1:8125
+      output_format:
+        type: string
+        example: Splunk
+      metrics:
+        $ref: "#/definitions/Metrics"
+  MetricFormats:
+    type: array
+    items:
+      type: object
+      properties:
+        output_format:
+          type: string
+          description: For customized metrics (which are different than the format defaults), use "custom" format
+          example: splunk
+        default_metrics:
+          $ref: "#/definitions/Metrics"
+  Metrics:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+        pattern:
+          type: string
+    example:
+      - name: Views
```

Comment:
what does name: Views mean?

Reply:
not sure i got you. That's an example of metric structure.

Reply:
Anyway, i added a description.